bin/telemetry_sim_ComCam

```diff
@@ -125,6 +140,8 @@ async def amain():
     dome = salobj.Controller(name="MTDome", index=0)
     rotator = salobj.Controller(name="MTRotator", index=0)
     dometraj = salobj.Controller(name="MTDomeTrajectory", index=0)
+    dimm = salobj.Controller(name="DIMM", index=1)
```

Comment:
Just checking: we don't have separate DIMM/WeatherStation information (at a different index) for the Simonyi Telescope versus the AuxTel?

Reply:
Will double check with folks.